src/cascadia/TerminalSettingsEditor/Profiles_Advanced.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "pch.h"
+#include "Profiles_Advanced.h"
+#include "Profiles_Advanced.g.cpp"
+
+#include "EnumEntry.h"
+#include <LibraryResources.h>
+#include "..\WinRTUtils\inc\Utils.h"
+
+using namespace winrt::Windows::UI::Xaml::Navigation;
+
+namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
+{
+    Profiles_Advanced::Profiles_Advanced()
+    {
+        InitializeComponent();
+
+        INITIALIZE_BINDABLE_ENUM_SETTING(AntiAliasingMode, TextAntialiasingMode, winrt::Microsoft::Terminal::Control::TextAntialiasingMode, L"Profile_AntialiasingMode", L"Content");
```

Comment:
Are these supposed to be here? These are in `ProfileViewModel` right now...

Reply:
Thanks for the catch! Yeah these don't need to be here